resources/views/home/_search_result_team.blade.php

```diff
@@ -0,0 +1,71 @@
+{{--
```

Comment:
actually, why isn't this React and `leader` added to the transformer include?

Reply:
🤷 it's just simpler/faster for initial work. maybe later when it's needed somewhere else